debs/SPECS/5.0.0/wazuh-agent/debian/changelog

```diff
@@ -1,50 +1,50 @@
 wazuh-agent (5.0.0-RELEASE) stable; urgency=low
 
-  * More info: https://documentation.wazuh.com/current/release-notes/
+ * More info: https://documentation.wazuh.com/current/release-notes/
 
- -- Wazuh, Inc <info@wazuh.com>  Sat, 04 Dec 2021 14:04:26 +0000
+-- Wazuh, Inc <info@wazuh.com>  Sat, 04 Dec 2021 14:04:26 +0000
  
 wazuh-agent (4.3.0-RELEASE) stable; urgency=low
 
-  * More info: https://documentation.wazuh.com/current/release-notes/
+ * More info: https://documentation.wazuh.com/current/release-notes/
 
- -- Wazuh, Inc <info@wazuh.com>  Wed, 28 Apr 2021 12:15:57 +0000
+-- Wazuh, Inc <info@wazuh.com>  Wed, 28 Apr 2021 12:15:57 +0000
 
 wazuh-agent (4.2.0-RELEASE) stable; urgency=low
 
-  * More info: https://documentation.wazuh.com/current/release-notes/
+ * More info: https://documentation.wazuh.com/current/release-notes/
 
- -- Wazuh, Inc <info@wazuh.com>  Mon, 26 Apr 2021 11:51:55 +0000
+-- Wazuh, Inc <info@wazuh.com>  Mon, 26 Apr 2021 11:51:55 +0000
 
 wazuh-agent (4.1.5-RELEASE) stable; urgency=low
 
-  * More info: https://documentation.wazuh.com/current/release-notes/
+ * More info: https://documentation.wazuh.com/current/release-notes/
 
- -- Wazuh, Inc <info@wazuh.com>  Thu, 22 Apr 2021 16:50:05 +0000
+-- Wazuh, Inc <info@wazuh.com>  Thu, 22 Apr 2021 16:50:05 +0000
 
 wazuh-agent (4.1.4-RELEASE) stable; urgency=low
 
-  * More info: https://documentation.wazuh.com/current/release-notes/
+ * More info: https://documentation.wazuh.com/current/release-notes/
 
- -- Wazuh, Inc <info@wazuh.com>  Mon, 29 Mar 2021 16:23:09 +0000
+-- Wazuh, Inc <info@wazuh.com>  Mon, 29 Mar 2021 16:23:09 +0000
 
 wazuh-agent (4.1.3-RELEASE) stable; urgency=low
 
   * More info: https://documentation.wazuh.com/current/release-notes/
 
- -- Wazuh, Inc <info@wazuh.com>  Sat, 20 Mar 2021 13:41:26 +0000
+-- Wazuh, Inc <info@wazuh.com>  Sat, 20 Mar 2021 13:41:26 +0000
 
 wazuh-agent (4.1.2-RELEASE) stable; urgency=low
 
-  * More info: https://documentation.wazuh.com/current/release-notes/
+ * More info: https://documentation.wazuh.com/current/release-notes/
 
- -- Wazuh, Inc <info@wazuh.com>  Mon, 08 Mar 2021 14:00:25 +0000
+-- Wazuh, Inc <info@wazuh.com>  Mon, 08 Mar 2021 14:00:25 +0000
 
 wazuh-agent (4.1.1-RELEASE) stable; urgency=low
 
-  * More info: https://documentation.wazuh.com/current/release-notes/
+ * More info: https://documentation.wazuh.com/current/release-notes/
```

Comment:
Fix tabulations please
![image](https://user-images.githubusercontent.com/10661307/123629648-cb7f5f80-d814-11eb-9e34-34c8fe93ccf1.png)

Reply:
Done in b335d2b